components/eamxx/src/physics/mam/eamxx_mam_aci_process_interface.cpp

```diff
@@ -506,6 +506,7 @@ void MAMAci::run_impl(const double dt) {
       coltend_, coltend_cw_, qcld_, ndropcol_, ndropmix_, nsource_, wtke_, ccn_,
       // ## output to be used by the other processes ##
       qqcw_fld_work_, ptend_q_, factnum_, tendnd_,
+      top_lev_,
```

Comment:
Could we move `top_lev_` in as part of the following block?
```
 team_policy, dt, dry_atm_, rpdel_, kvh_mid_, kvh_int_, wsub_, cloud_frac_,
 cloud_frac_prev_, dry_aero_, nlev_, enable_aero_vertical_mix_,
```
Although not labeled, this block represent the input.